backend/apps/models.py

```diff
@@ -1,3 +1,308 @@
+    name = models.CharField(max_length=255)
+    tagline = models.CharField(max_length=255)
+    application_id: models.IntegerField
+    social_accounts: ArrayField(models.CharField(max_length=255))
+    total_flags: models.IntegerField
+    created_by: models.IntegerField
+    creation_date: models.DateField
+    class Meta:
+        verbose_name = 'organization'
+        verbose_name_plural = '2. Organizations'
+    def __str__(self):
+        return self.name
+    
+class Event(models.Model):
+    creation_date = models.DateField
+    created_by = models.ForeignKey(User, on_delete=models.CASCADE)
+    name = models.CharField(max_length=255)
+    tagline = models.CharField(max_length=255)
+    start_time = models.DateField
+    end_time = models.DateField
+    type = models.CharField(max_length=255)
+    format = models.CharField(max_length=255)
+    online_location_link = models.CharField(max_length=255)
+    offline_location_name = models.CharField(max_length=255)
+    offline_location_lat = models.FloatField
+    offline_location_long = models.FloatField
+    description = models.TextField(max_length=500)
+    get_involved_text = models.TextField(max_length=500)
+    deletion_date = models.DateField
+
+class Role(models.Model):
+    name = models.CharField(max_length=255)
+    is_custom = models.BooleanField
+    creation_date = models.DateField
+    description = models.TextField(max_length=500)
+    last_updated = models.DateField
+    deprecation_date = models.DateField     
+    class Meta:
+        verbose_name = 'role'
+        verbose_name_plural = '1. Role'
+    def __str__(self):
+        return self.name
+    
+class Topic(models.Model):
+    name = models.CharField(max_length=255)
+    active = models.BooleanField
+    creation_date = models.DateField
+    description = models.TextField(max_length=500)
+    last_updated = models.DateField
+    deprecation_date = models.DateField
+    class Meta:
+        verbose_name = 'topic'
+        verbose_name_plural = '1. Topic'
+    def __str__(self):
+        return self.name
+    
+class Resource(models.Model):
+    name = models.CharField(max_length=255)
+    topics = ArrayField(models.CharField(max_length=255)) 
+    location = models.CharField(max_length=255)
+    url = models.CharField(max_length=255)
+    total_flags = models.IntegerField
+    creation_date = models.DateField
+    description = models.TextField(max_length=500)
+    last_updated = models.DateField
+    deletion_date = models.DateField
+    class Meta:
+        verbose_name = 'resource'
+        verbose_name_plural = '1. Resource'
+    def __str__(self):
+        return self.name
+    
+class Format(models.Model):
+    name = models.CharField(max_length=255)
+    creation_date = models.DateField
+    description = models.TextField(max_length=500)
+    last_updated = models.DateField
+    deprecation_date = models.DateField
+    class Meta:
+        verbose_name = 'format'
+        verbose_name_plural = '1. Format'
```

Comment:
Could you let me know why we're doing `1./2. CAP_VERBOSE_NAME` for the `verbose_name_plural`, @momanyisamuel? Is this something to format it for the admin panel?

Reply:
yes this is for the admin panel